requirements-dev.txt

```diff
@@ -6,9 +6,6 @@ pdbpp==0.10.2
 psutil==5.9.0
 requests==2.27.1
 
-# linting
-pre-commit>=2.2.0
```

Comment:
We are currently managing this version in two places – here and in the `.pre-commit-config.yaml` file – and they already diverged. There is no reason to include it here, users who wish to use it can install it on their own (in any way they like), and GitHub Actions uses the pre-commit action now which is versioned separately.

Reply:
I had to remove the action, because it is broken with local hooks, and I had to add `filelock` as direct dependency, because it was used indirectly. Still, the sentiment towards having this dependency stays the same on my side.